packages/react-composites/src/composites/CallComposite/StartCallButton.tsx

```diff
@@ -26,7 +27,7 @@ export const StartCallButton = (props: StartCallButtonProps): JSX.Element => {
     <PrimaryButton
       data-ui-id="call-composite-start-call-button"
       disabled={isDisabled}
-      className={buttonStyle}
+      className={css(buttonStyle, props.className)}
```

Comment:
why not using 'mergeStyles' ?

Reply:
They are surprisingly identical, one must be a legacy function, updating to mergeStyles to match the rest of the codebase